app/models/motion.rb

```diff
@@ -47,7 +47,7 @@ class Motion < ActiveRecord::Base
   scope :lapsed_but_not_closed,    -> { voting.lapsed }
   scope :closed,                   -> { where('closed_at IS NOT NULL').order(closed_at: :desc) }
   scope :order_by_latest_activity, -> { order('last_vote_at desc') }
-  #scope :visible_to_public,        -> { joins(:discussion).merge(Discussion.public) }
```

Comment:
Ah, I had just assumed that was garbage since it was commented out and `Discussion.public` wasn't a thing. I'll swap it to merge and use `Discussion.visible_to_public` instead